src/components/NewCommentForm.tsx

```diff
@@ -1,8 +1,76 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { FormEvent, useState } from 'react';
+import { CommentData } from '../types/Comment';
+
+type Props = {
+  addComment: (
+    newComment: CommentData,
```

Comment:
Consider renaming 'offButtonLoad' to 'setButtonLoading' for better readability and consistency with other state setters.